core/modules/views/plugins/views_plugin_pager_mini.inc

```diff
@@ -26,7 +26,10 @@ class views_plugin_pager_mini extends views_plugin_pager_full {
   function option_definition() {
     $options = parent::option_definition();
 
-    unset($options['quantity']);
+    // Intentionally setting $options['quantity'] to NULL here instead of using
+    // unset(), otherwise it throws "undefined index" notices.
+    // See: https://github.com/backdrop/backdrop-issues/issues/1435
```

Comment:
Not sure we need to link to issues each time we make a change in core...

If we do decide this link is required here, I'd suggest the following syntax instead:
```suggestion
    // @see https://github.com/backdrop/backdrop-issues/issues/1435
```

Reply:
I thought that `@see` was reserved for pointing to code, but you are absolutely right @BWPanda 👍🏼 

@see (pun 😅 ): https://www.drupal.org/docs/develop/standards/api-documentation-and-comment-standards#see

Reply:
...as for adding issue links, I think that this is only needed if there is more to it (i.e. a few words in the docblock are not enough to explain what's happening). Happy to remove it entirely (I believe that the core committer that merges this can take care of that - if not, then let me know, and I'll do it).